QUICKSTART.md

```diff
@@ -0,0 +1,68 @@
+# Prerequisites
+1. OpenSSL 3.0 or higher, for cryptographic key generation, and the C API.
+2. Java 11 or higher, for *Auth*.
+3. NodeJS, Npm, and Maven.
+5. CMake 3.19 or higher, for C example compilation.
+## Ubuntu 24.04
+```
+$ apt-get install openssl openjdk-17-jdk nodejs npm maven cmake
+
+$ git clone https://github.com/iotauth/iotauth.git
+$ cd iotauth
+$ git submodule update --init
```

Comment:
These three lines are common for both Ubuntu and Mac, thus can be merged into one.

Reply:
Added another section as `Clone repository & Update submodule`